src/main/groovy/com/bmuschko/gradle/docker/DockerRemoteApiPlugin.groovy

```diff
@@ -29,7 +29,7 @@ import org.gradle.api.file.FileCollection
  */
 class DockerRemoteApiPlugin implements Plugin<Project> {
     public static final String DOCKER_JAVA_CONFIGURATION_NAME = 'dockerJava'
-    public static final String DOCKER_JAVA_DEFAULT_VERSION = '3.0.6'
+    public static final String DOCKER_JAVA_DEFAULT_VERSION = '3.0.7'
```

Comment:
Is a new version of Docker Java required for supporting labels? I'd prefer a new PR for the change of the Docker Java library if it is not required.